SDL/Quintuplet.cuh

```diff
@@ -998,7 +998,7 @@ namespace SDL
         float sdlPVoff = 0.1f/rt_OutLo;
         sdlCut = alpha1GeV_OutLo + alpaka::math::sqrt(acc, sdlMuls * sdlMuls + sdlPVoff * sdlPVoff);
 
-        deltaPhiPos = SDL::deltaPhi_alpaka(acc, mdsInGPU.anchorX[secondMDIndex], mdsInGPU.anchorY[secondMDIndex], mdsInGPU.anchorX[fourthMDIndex], mdsInGPU.anchorY[fourthMDIndex]);
+        deltaPhiPos = SDL::deltaPhi(acc, mdsInGPU.anchorX[secondMDIndex], mdsInGPU.anchorY[secondMDIndex], mdsInGPU.anchorX[fourthMDIndex], mdsInGPU.anchorY[fourthMDIndex]);
```

Comment:
It seems to me that there is some inconsistency on how we use the `SDL` namespace for these functions. I guess it should always be explicitly there for the `copysignf` function, since there is a one with the same name in the standard `math.h` library but I understand that it doesn't have to be there for the `deltaPhi` function, since it isn't there in other files (contrary to what is done here). Should we try to homogenize?

For my own understanding, is it true that the `SDL` namespace is mandatory for the `copysignf`? Is this the case for other functions?

Reply:
Yeah that's a good point, I think best practice would be to add the SDL prefix to the relevant functions so that it's clear  where these functions are being called from. I agree that copysignf is the only function where it's really required, but good either way to homogenize everything like you said.